lib/rules/no-multi-spaces.js

```diff
@@ -99,3 +99,21 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.schema = [
+    {
+        "type": "object",
+        "properties": {
+            "exceptions": {
+                "type": "object",
+                "patternProperties": {
+                    "^([A-Z][a-z]*)+$": {
```

Comment:
Wait, mine is not a check for pascal case. It's just a check for a string of letters. Yours is a check for PascalCased string. Not 100% sure it's necessary, but those two RegEx do different things.